Detectors/TPC/calibration/SpacePoints/src/ResidualAggregator.cxx

```diff
@@ -182,7 +205,7 @@ bool ResidualAggregator::hasEnoughData(const Slot& slot) const
 
 void ResidualAggregator::initOutput()
 {
-  // nothing to be done here, but method needs to be overwritten
+  mFileMetaData = std::make_unique<o2::dataformats::FileMetaData>();
```

Comment:
Why do you need this? It is anyway assigned at L227. You could actually use a local FileMetaData object at that line, w/o any smart pointers. I will also change this in the CTFWriter.

Reply:
Sorry I did not really think about it when I took that from the CTF writer.. It's changed now to a local object